source/frontend/src/components/common/mapFSM/MapStateMachineContext.tsx

```diff
@@ -384,20 +401,22 @@ export const MapStateMachineProvider: React.FC<React.PropsWithChildren<unknown>>
         showPopup: showPopup,
         isLoading: state.context.isLoading,
         mapSearchCriteria: state.context.searchCriteria,
+        advancedSearchCriteria: state.context.advancedSearchCriteria,
         mapFeatureData: state.context.mapFeatureData,
         filePropertyLocations: state.context.filePropertyLocations,
         pendingFitBounds: state.matches({ mapVisible: { mapRequest: 'pendingFitBounds' } }),
         requestedFitBounds: state.context.requestedFitBounds,
         isSelecting: state.matches({ mapVisible: { featureView: 'selecting' } }),
         isRepositioning: isRepositioning,
         selectingComponentId: state.context.selectingComponentId,
-        isFiltering: state.context.isFiltering,
+        isFiltering: !dequal(state.context.advancedSearchCriteria, new PropertyFilterFormModel()),
         isShowingMapFilter: isShowingMapFilter,
         isShowingMapLayers: isShowingMapLayers,
         activeLayers: state.context.activeLayers,
         activePimsPropertyIds: state.context.activePimsPropertyIds,
         showDisposed: state.context.showDisposed,
         showRetired: state.context.showRetired,
+        isMapVisible: state.matches({ mapVisible: {} }),
```

Comment:
required so that the MapContainer doesn't sent the list of visible map properties before the map is loaded, meaning the state machine ignores the event.